microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -980,12 +1014,41 @@ Selecting this experiment checkbox adds the **Preloading** section within the **
 
 <!--
 Select an element for more details.
-todo: how select element, where?  that instruction is prominent in UI, so clarify here
+todo old: how select element, where?  that instruction is prominent in UI, so clarify here
 -->
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is present in Microsoft Edge Stable 118.
+* This checkbox is not present in Microsoft Edge Canary 121.
+* This checkbox is present in Microsoft Edge Stable 119.
+
+
+<!-- ====================================================================== -->
+## Enable Speculative Loads Panel in Application panel
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 121_
```

Comment:
3. @captainbrosset @zoherghadyali - need info about the Experiments checkbox:
Enable Speculative Loads Panel in Application panel

Reply:
The [Speculation Rules API](https://github.com/jeremyroman/alternate-loading-modes/blob/main/triggers.md#speculation-rules) can be used to programmatically tell Edge to prerender certain pages. This is useful for web developers because pre-rendered pages are near instant to navigate to (the browser pre-renders, in a hidden way, the next pages that a user is likely to navigate to, while the user is still on the current page).

The experiment enables the new page within the Application tool. The new page lists the speculation rules that the web author has defined in their webpage. And also gives debugging information that lets them see whether a rule was actually used or not (i.e. whether a particular page was pre-rendered or not).